modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/resources/META-INF/resources/alloyeditor.jsp

```diff
@@ -81,7 +81,13 @@ if (editorOptions != null) {
 <liferay-util:buffer
 	var="alloyEditor"
 >
-	<div class="alloy-editor alloy-editor-placeholder <%= HtmlUtil.escapeAttribute(cssClass) %>" contenteditable="false" data-placeholder="<%= LanguageUtil.get(request, placeholder) %>" data-required="<%= required %>" id="<%= HtmlUtil.escapeAttribute(name) %>" name="<%= HtmlUtil.escapeAttribute(name) %>"></div>
+	<div class="alloy-editor <%= HtmlUtil.escapeAttribute(cssClass) %>" contenteditable="false" data-placeholder="<%= LanguageUtil.get(request, placeholder) %>" data-required="<%= required %>" id="<%= HtmlUtil.escapeAttribute(name) %>" name="<%= HtmlUtil.escapeAttribute(name) %>"></div>
+	<div class="alloy-editor-placeholder <%= HtmlUtil.escapeAttribute(cssClass) %>">
+		<%= LanguageUtil.get(request, placeholder) %>
+		<c:if test="<%= Boolean.parseBoolean(required) %>">
```

Comment:
This placeholder looks fixed. I think it should be removed once the editor content is defined

https://user-images.githubusercontent.com/7663513/140067143-f2eb6c20-fc0a-4e6e-bdb8-65b6f865dbb8.mov

I deployed *alloyeditor-web, *ckeditor-web, blogs-web, *styled modules. May I missed something?

Reply:
When changing things inside frontend-theme-styled module it's necessary re-deploy the entire frontend-theme module. When deploying it, it works now :)